CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# 1.21.0 (30.08.2017)
+- [Allow custom CLI class instances](https://github.com/serverless/serverless/pull/4160)
+- [Add support in Spotinst Functions](https://github.com/serverless/serverless/pull/4127)
+- [Add PHP support for OpenWhisk](https://github.com/serverless/serverless/pull/4153)
+- [Fixed a bug with stack deletion monitoring](https://github.com/serverless/serverless/pull/4132)
+- [Allow AWS Profile CLI option to overwrite config and env](https://github.com/serverless/serverless/pull/3980)
+- [Improve performance of the package plugin](https://github.com/serverless/serverless/pull/3924)
+- [Add support for custom context with Invoke Local](https://github.com/serverless/serverless/pull/4126)
+- [Add aws-nodejs-typescript template](https://github.com/serverless/serverless/pull/4058)
+- [Add aws-nodejs-ecma-script template](https://github.com/serverless/serverless/pull/4056)
+- [Allow updates for AWS profiles](https://github.com/serverless/serverless/pull/3866)
+- [Allow updates for AWS profiles](https://github.com/serverless/serverless/pull/3866)
```

Comment:
You got this in here twice...

Reply:
Thanks for pointing that out @lorengordon 👍

Will be fixed with https://github.com/serverless/serverless/pull/4181.